Added a branding and image customization example to the app. (#15)
Salesforce-Async-Messaging/messaging-in-app-ios: README.md

- [Android Messaging for In-App GitHub Repo](https://github.com/Salesforce-Async-Messaging/messaging-in-app-android)

![iOS Device](./images/messaging-ios-device.png)

## Customization: 

> This app provides an example of how you can customize it to replace images and override branding. For instance, the actionMenu image in the chat feed has been replaced, and the branding for error messages has been changed.

Reviewer: ```suggestion
This app provides an example of how you can customize images and override branding. Specifically, the `actionMenu` image in the chat feed has been replaced, and the color branding for error messages has been changed.
```
